contracts/libraries/Errors.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+/**
+ * @title Errors library
+ * @author Cruize
+ * @notice Defines the error messages emitted by the different contracts of the Cruize protocol
+ */
+library Errors {
```

Comment:
can you create custom errors? it would be much better.
take a look at this site for custom  error handling in solidity https://blog.soliditylang.org/2021/04/21/custom-errors/